Adjust_ACIS-Continuity.py

```diff
@@ -0,0 +1,130 @@
+
+import argparse
+import shutil
+
+import apt_date_secs as apt
+#import BackstopCommandsClass
+from backstop_history import BackstopHistory
+"""
+Adjust_ACIS-Continuity.py - LR has determined that a continuity load was reviewed as a full load,
+                                              but was uplinked and run VEHICLE ONLY because the load prior to
+                                              the Continuity load was interrupted by either a 107-only or a
+                                              Full Stop AFTER the Continuity load was reviewed and approved.
+
+                                              Therefore the Continuity load's ACIS-Continuity.txt file needs to
+                                              be updated with the interrupt type and the time of interrupt. This
+                                              information is available in the NLET file.
+
+                                              Also important is the fact that this adjustment need only be done once:
+                                              when the Review load is the A load. So this routine is only called for A loads.
+
+                                              The NLET file entries that matter are either "S107" or "STOP"
+                                              The ACIS-Continuity.txt file adjustment will be SCS-107  for S107
+                                              or STOP for Full Stops.
+
+The graphical timeline representation of events:
+
+| Continuity-Continuity Load|    
+             |
+      Interrupt
+
+                      | Continuity load built, Reviewed and Approved prior to the interrupt |
+
+                                                                                                               |    Review Load |                                          
+"""
+# Create a parser instance
+myparser = argparse.ArgumentParser()
+
+# Adding  switches that are both REQUIRED and POSITIONAL
+
+myparser.add_argument("rev_load_ofls_dir", help="Full path to the Review load Directory.")
+
+myparser.add_argument("cont_load_dir", help="Full path to the Continuity load directory.")
+
+myparser.add_argument("nlet_file_path", help="Full path to the NLET file.")
+
+args = myparser.parse_args()
+
+# Create an array of event types to recognize
+event_types =  ["S107", "STOP", "VO_SCS-107"]
+
+# Capture the command line arguments into variables.
+rev_load_ofls_dir = args.rev_load_ofls_dir
+cont_load_dir = args.cont_load_dir
+nlet_file_path =  args.nlet_file_path
+      
+# Capture the Review and Continuity load week names
+rev_load_week = rev_load_ofls_dir[28:35]
```

Comment:
When I ran the test loads I did not run it as -T.  Instead I created ficticious loads (MAY1524 and MAY1624) from the MAY1324 and MAY1424 loads.  I had to avoid -T loads because using -T deactivates some of the code that had to be tested.

Reply:
But yes I see what you are saying, on second look....I have to make an adjustment here still.

Reply:
Code fixed.